doc/ad_code_values/biden/12/NB.md

```diff
@@ -0,0 +1,10 @@
+---
+value: NB
+name: New Battleground States
+confidence: medium
```

Comment:
I've downcased this, as I've noticed it wouldn't show up with the yellow confidence badge with a capital M :)